build/buildorder.props

```
@@ -8,18 +8,18 @@
 
   <ItemGroup>
     <RepositoryBuildOrder Include="Razor" Order="6" />
-    <RepositoryBuildOrder Include="HttpSysServer" Order="8" />
+    <RepositoryBuildOrder Include="HttpSysServer" Order="8" RootPath="$(RepositoryRoot)src\HttpSysServer\" />
     <RepositoryBuildOrder Include="BasicMiddleware" Order="9" />
-    <RepositoryBuildOrder Include="Antiforgery" Order="10" />
+    <RepositoryBuildOrder Include="Antiforgery" Order="9" RootPath="$(RepositoryRoot)src\Antiforgery\" />
```

```suggestion
    <RepositoryBuildOrder Include="Antiforgery" Order="10" RootPath="$(RepositoryRoot)src\Antiforgery\" />
```

Just to be safe, avoid build order changes.